Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ruby:3.2
+
+WORKDIR /app
+COPY . .
+
+RUN gem install bundler jekyll
```

Comment:
No point in this as its already in the Gemfile.lock and could install a different versions than we're really using.

```suggestion
```